test/rotoscope_test.rb

```diff
@@ -317,6 +331,18 @@ def test_trace_uses_io_objects
     assert_frames_consistent contents
   end
 
+  def test_stop_trace_before_start_does_not_raise
+    rs = Rotoscope.new(@logfile)
+    rs.stop_trace
+  end
+
+  def test_gc_rotoscope_without_stop_trace
+    rs = Rotoscope.new(@logfile)
+    rs.start_trace
+    rs = nil
+    GC.start
+  end
```

Comment:
To avoid adding a flaky test, I'm going to add a GC.start at the start of this test so it is still a regression test for the issue fixed in this PR.  I'll work on another PR to fix this additional crash.

I'm thinking we could avoid the problem by copying the blacklist into `malloc`ed memory so we can make sure it isn't freed before the Rotoscope object is freed.